.rhcicd/build_deploy.yml

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="notifications"
```

Comment:
I think we need to update this to `user-preferences` since that's what the image will end up being named.